Update format-output-azure-cli.md (#3965)
MicrosoftDocs/azure-docs-cli: docs-ref-conceptual/format-output-azure-cli.md

`none`   | No output other than errors and warnings

> [!WARNING]

> > Use an output format of `none` or store command output in a variable to avoid exposing secrets such as API keys and credentials.

Reviewer: ```suggestion
> Use an output format of `none` or store command output in a variable to avoid exposing secrets such as API keys and credentials. **Note:** Certain CI/CD environments may store the output of the executed commands in logs. It is a good practice to confirm what is written in those log files and who has access to the logs.
```

- secrets
- keys

To protect secrets and keys when using Azure CLI commands, choose one of these options:

|Option|Benefit|Use case|
|-|-|-|

> |`--output none` output format| Keeps security information from displaying in your console. If your command fails, you'll still receive error messages.| 1. Use when command output _can be_ retrieved at a later time.|

Reviewer: ```suggestion
|`--output none` output format| Keeps sensitive information from being displayed in your console. If your command fails, you'll still receive error messages.| 1. Use when command output _can be_ retrieved at a later time.|
```

Reset a service principal credential returning output in the default json format:

```azurecli-interactive

> # reset service principal credentials returning.

Reviewer: ```suggestion
# reset service principal credentials using default output format (json).
```

}
```

> A better solution is to return security information to a variable.

Reviewer: ```suggestion
A better solution is to return sensitive information to a variable.
```
